lib/cdo/script_constants.rb

```diff
@@ -74,10 +74,16 @@ module ScriptConstants
     ],
     hoc: [
       # Note that now multiple scripts can be an 'hour of code' script.
-      # If adding a script here,
-      # you must also update the cdo-tutorials gsheet so the end of script API works
+      # If adding a script here, you must also update the cdo-tutorials gsheet
+      # so the end of script API works; specifically, there needs to be a row
+      # with code_s matching the script name (in quotes) in this list.
 
       nil,
+      POEM_ART_2021_NAME = 'poem-art-2021'.freeze, # 2021 hour of code
+      HELLO_WORLD_FOOD_2021_NAME = 'hello-world-food-2021'.freeze, # 2021 hour of code
+      HELLO_WORLD_ANIMALS_2021_NAME = 'hello-world-animals-2021'.freeze, # 2021 hour of code
+      HELLO_WORLD_EMOJI_2021_NAME = 'hello-world-emoji-2021'.freeze, # 2021 hour of code
+      HELLO_WORLD_RETRO_2021_NAME = 'hello-world-retro-2021'.freeze, # 2021 hour of code
```

Comment:
@breville @KylieModen This is only 5 of the 10 scripts I was askd to copy over. Is that correct? Do we want the other new scripts in the list too?

Reply:
The other scripts are CSC (not HoC) so they don’t need this treatment

Reply:
Resolved by adding a new CSC section